app/pages/system/SiloImagesPage.tsx

```diff
@@ -52,12 +53,26 @@ SiloImagesPage.loader = async () => {
 }
 
 export function SiloImagesPage() {
-  const { Table, Column } = useQueryTable('imageList', {})
+  const { Table } = useQueryTable('imageList', {})
   const [showModal, setShowModal] = useState(false)
 
   const [demoteImage, setDemoteImage] = useState<Image | null>(null)
 
   const queryClient = useApiQueryClient()
+
+  const colHelper = createColumnHelper<Image>()
+  const staticCols = [
+    colHelper.accessor('name', {
+      cell: makeLinkCell((image) => pb.siloImageEdit({ image })),
+    }),
+    colHelper.accessor('description', {}),
+    colHelper.accessor('size', { cell: (info) => <SizeCell value={info.getValue()} /> }),
+    colHelper.accessor('timeCreated', {
+      header: 'Created',
+      cell: (info) => <DateCell value={info.getValue()} />,
+    }),
+  ]
```

Comment:
In the interest of not doing things inside the render loop when we can avoid it, cases like these where all the columns are fully static should be moved outside the component. This is true here and for projects, sleds, and floating IPs.